### PR TITLE
Const constructor audit.

### PR DESCRIPTION
**Pull request by Hixie**
Where possible this uses this boilerplate text instead of bespoke text
that doesn't say anything more useful:

```
  /// Abstract const constructor. This constructor enables subclasses to provide
  /// const constructors so that they can be used in const expressions.
```

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by Hixie** [comment]
cc @HansMuller

File: packages/flutter/lib/src/widgets/actions.dart [hunk 1]
```diff
@@ -1561,7 +1562,7 @@ class DoNothingAction extends Action<Intent> {
 ///  * [WidgetsApp.shortcuts], which defines the shortcuts to use in an
 ///    application (and defaults to [WidgetsApp.defaultShortcuts]).
 class ActivateIntent extends Intent {
-  /// Creates a const [ActivateIntent] so subclasses can be const.
+  /// Creates ain intent that activates the currently focused control.
```
Comment (HansMuller) [inline packages/flutter/lib/src/widgets/actions.dart:1565]: ain => an

File: packages/flutter/lib/src/widgets/preferred_size.dart [hunk 2]
```diff
@@ -111,7 +111,7 @@ abstract class PreferredSizeWidget implements Widget {
 ///    its preferred size.
 ///  * [AppBar] and [TabBar], which implement PreferredSizeWidget.
 class PreferredSize extends StatelessWidget implements PreferredSizeWidget {
-  /// Creates a widget that has a preferred size.
+  /// Creates a widget that has a preferred size that the parent can query.
```
Comment (HansMuller) [inline packages/flutter/lib/src/widgets/preferred_size.dart:114]: the parent => any ancestor

**Review by HansMuller — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/widgets/actions.dart (hunk 1 above)
Comment (Hixie) [reply packages/flutter/lib/src/widgets/actions.dart:1565]: good catch, fixed

File: packages/flutter/lib/src/widgets/preferred_size.dart (hunk 2 above)
Comment (Hixie) [reply packages/flutter/lib/src/widgets/preferred_size.dart:114]: How would someone other than a parent query it?

Comment (Hixie) [reply packages/flutter/lib/src/widgets/preferred_size.dart:124]: i've added some more docs here but i'm pretty sure "parent" is actually right here.

**Comment by Hixie** [comment]
Landing on green, please let me know if I'm wrong about the parent/ancestor thing and I'll fix in a subsequent PR. Thanks!

File: packages/flutter/lib/src/widgets/preferred_size.dart (hunk 2 above)
Comment (HansMuller) [reply packages/flutter/lib/src/widgets/preferred_size.dart:124]: My mistake, parent is fine.

**Pull request merged**